e2e/setup-local-fdb-cluster.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+# Set up local FDB non-HA cluster on a 4-node kind k8s cluster
+
+SkipCreatingLocalCluster=${1:-0}
+
+cluster="local-cluster"
+
+if ${SkipCreatingLocalCluster} != 0; then
+    echo "===Start creating k8s cluster on kind"
+    kind create cluster --name ${cluster} --config ./local-cluster-config.yaml
+else
+    echo "===Skip creating k8s cluster on kind"
+    kind get clusters
```

Comment:
Should we ensure the current context is pointing to the kind cluster? Otherwise what is the benefit of doing this get command?

Reply:
"Should we ensure the current context is pointing to the kind cluster?" <-- It will be good to do that. I didn't do that because I assume this is a clean install with kind. 
` kind get clusters` is just a hint that in case kind is pointing to a wrong context, the clusters will be different, which hints that user should change the context

Reply:
I think for now it's okay to leave it as it is, but could you please add a TODO above this statement?